src/Mono.Android/metadata

```diff
@@ -1390,16 +1387,10 @@
 
   We will have to enable this line and bring back some sanity.
   -->
-  <!-- attr path="/api/package[@name='android.graphics']/class[@name='Color']" name="managedName">AndroidColor</attr -->
+  <attr path="/api/package[@name='android.graphics']/class[@name='Color']" name="managedName">ColorObject</attr>
```

Comment:
Why `ColorObject` and not `AndroidColor`?

Reply:
I am 100% sure that people try to look up "Color" something. AndroidColor will never give that impression to developers.